modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx

```diff
@@ -44,27 +44,35 @@ import {IDataSet} from '../../DataSets';
 import RequiredMark from '../../components/RequiredMark';
 import sortItems from '../../utils/sortItems';
 
+const FILTERS_ORDER_OBJECT_FIELD_NAME = Liferay.FeatureFlags['LPD-15729']
```

Comment:
@chestofwonders this affects https://github.com/liferay-frontend/liferay-portal/pull/4208